README.md

```diff
@@ -6,16 +6,40 @@ You can use it to display a custom cookies message of your choosing on your own
 
 ## Usage
 
-This project can be installed from an [NPM package](https://www.npmjs.com/package/cookie-policy).
+This project can be installed via an [NPM package](https://www.npmjs.com/package/cookie-policy).
+
+```bash
+yarn add cookie-policy
+npm install cookie-policy --save
+```
+
+You can then install the library either by directly linking to it or via an ES6 import.
+
+1. Via direct link
+
+To consume the library directly, add a link to the JS file containing an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) and call the lib:
+
+```javascript
+<script src="/node_modules/cookie-policy/build/js/cookie-policy.js"></script>
```

Comment:
`node_modules` should be outside of the static paths of a webserver for security reasons so we should encourage people to copy the asset if they will not be importing the package directly.